WIP-CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - Bump falco-exporter chart to v0.8.0.
 - Users are now not forced to use proxy for connecting to alertmanager but can use port-forward as well.
 - The OpenSearch security config will now be managed completely by securityadmin
+- Patched Falco rules for `Change thread namespace` & `System procs network activity` as per our apps.
```

Comment:
I think it would be easier to understand if you just wrote something like:

```suggestion
- Patched Falco rules and added the rules `Change thread namespace` & `System procs network activity`.
```

Reply:
Thanks fredrik !! Fixed